model/project_parser.go

```diff
@@ -514,7 +517,10 @@ func LoadProjectForVersion(v *Version, id string, shouldSave bool) (*Project, *P
 		return nil, nil, errors.New("version has no config")
 	}
 	p := &Project{}
-	pp, err = LoadProjectInto([]byte(v.Config), id, p)
+	// opts empty because project yaml that has include will hit this case
```

Comment:
sorry, I don't fully understand this comment. Do you mean because we won't have include?